### PR TITLE
question-34-added

### PR DESCRIPTION
**Pull request by RahulRana701**
I have added question number 34 based on object literals and update the readme file as well

Screenshot 2023-05-10 204520

### CODE REVIEW

**Comment by kirandash** [comment]
> hi @kirandash I have created a pr , hope you merge it 

Thanks, I will review it soon and merge

File: README.md [hunk 1]
```diff
@@ -1141,3 +1141,50 @@ Finally, the `parseInt()` function is used to convert the difference_in_days val
...
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### ANSWER: A
+
+The code defines an object literal person with properties such as name, age, hobbies, and address, and a method sayHello.
+
+The console.log() statements print the value of name, the second element of the hobbies array (which is "traveling"), and the value of the city property in the address object (which is "New York").
+
+Finally, the method sayHello is called on the person object using dot notation, which outputs the string "Hello, my name is John" to the console.
```
Comment (kirandash) [inline README.md:1185]: Suggested changes for better readability of code

```diff
- The code defines an object literal person with properties such as name, age, hobbies, and address, and a method sayHello.
- 
- The console.log() statements print the value of name, the second element of the hobbies array (which is "traveling"), and the value of the city property in the address object (which is "New York").
- 
- Finally, the method sayHello is called on the person object using dot notation, which outputs the string "Hello, my name is John" to the console.

+ The code defines an object literal `person` with properties such as `name`, `age`, `hobbies`, and `address`, and a method `sayHello`.
+ 
+ The `console.log()` statements print the value of `name`, the second element of the `hobbies` array (which is "traveling"), and the value of the `city` property in the `address` object (which is "New York").
+ 
+ Finally, the method `sayHello` is called on the `person` object using dot notation, which outputs the string "Hello, my name is John" to the console.
```

File: code/034/script.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+let person = {
```
Comment (kirandash) [inline code/034/script.js:1]: Please format the code indentation properly. 

Developer Tips: You can use [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) on vscode to auto format your code.

```diff
- let person = {
-     name: "John",
-     age: 30,
-     hobbies: ["reading", "traveling", "cooking"],
-     address: {
-       street: "123 Main St",
-       city: "New York",
-       state: "NY"
-     },
-     sayHello: function() {
-       console.log("Hello, my name is " + this.name);
-     }
-   };
-   
-   console.log(person.name); 
-   console.log(person.hobbies[1]); 
-   console.log(person.address.city); 
-   
-  
-   person.sayHello(); 

+ let person = {
+   name: "John",
+   age: 30,
+   hobbies: ["reading", "traveling", "cooking"],
+   address: {
+     street: "123 Main St",
+     city: "New York",
+     state: "NY",
+   },
+   sayHello: function () {
+     console.log("Hello, my name is " + this.name);
+   },
+ };
+ 
+ console.log(person.name);
+ console.log(person.hobbies[1]);
+ console.log(person.address.city);
+ 
+ person.sayHello();
```

Once formatted on code, please format the same on README file too

**Review by kirandash — CHANGES_REQUESTED** [review]
Hi @RahulRana701 Thanks for the PR! Everything is awesome. 

Just some minor suggestions regarding the formatting of code and README texts.

Feel free to let me know if you need any help in fixing these.

**Comment by RahulRana701** [comment]
hi @kirandash made the changes as you suggested...

**Comment by kirandash** [comment]
fixes #24

**Pull request merged**